lms/envs/aws.py

```diff
@@ -1099,6 +1099,18 @@
 INSTALLED_APPS.append('openedx.features.ucsd_features')
 
 ############### Settings for Caliper Tracking #####################
+# 'caliper_tracking' app enables us to transform Edx event-logs according to
+# IMSGlobal Caliper Standards.
+# https://www.imsglobal.org/sites/default/files/caliper/v1p1/caliper-spec-v1p1/caliper-spec-v1p1.html.
+#
+# Dependencies:
+#   1. This app adds its processor in both the event tracking engine's settings
+#      [EVENT_TRACKING_BACKENDS, TRACKING_BACKENDS]
+#   2. This app must be placed in INSTALLED_APPS settings before
```

Comment:
This line is unnecessary because the caliper_app already does this automatically. It only needs to be placed in installed apps.

Reply:
I wrote this to explain why our code is placing 'caliper_tracking' above 'eventtracking' app.